README.md

```diff
@@ -66,6 +67,12 @@ use any other group you desire.
 
 The duration (in milliseconds) to highlight changes. Default is 300.
 
+## highlight_for_count
+
+Whether it should highlight when a `<count>` is provided before the key or not.
```

Comment:
I would change the wording here:

Enable support for highlighting when when a `<count>` is provided before the key.
If set to `false` it will only highlight when the mapping is not prefixed with a `<count>`.
Default: `true`.

Reply:
Other than that it looks great.
Thank you.